docs/content/en/docs/clustermgmt/observability/cluster-verify.md

```diff
@@ -1,122 +1,44 @@
 ---
 title: "Verify cluster"
 linkTitle: "Verify cluster"
```

Comment:
Lets change `title` to "Verify EKS Anywhere cluster health" and `linkTitle` to "Verify cluster health"